clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -4782,9 +4782,12 @@ void SYCLIntegrationHeader::emit(raw_ostream &O) {
     O << "    return 0;\n";
     O << "#endif\n";
     O << "  }\n";
+    StringRef ReturnType =
```

Comment:
It might be useful to just put the typedef at the top of the int-header with the `nullptr_t` typedef (and a couple of others?) so that you can just use it?  WDYT?

Reply:
Oh!  I see what you mean now.   Sure, I can do that.

Reply:
I actually don't see where we do that, but I could have sworn Jeff was working on that at one point?  IF we ended up doing some of those typedefs that is perhaps my suggestion, otherwise this LGTM.

Reply:
I thought you meant add them anew, but my recollection of Jeff's attempt is that he had to abandon it.
I will leave it as is for now.